src/drafts/SelectPanel2/SelectPanel.tsx

```diff
@@ -303,6 +366,7 @@ const SelectPanelSearchInput: React.FC<TextInputProps> = ({onChange: propsOnChan
         <TextInput.Action
           icon={XCircleFillIcon}
           aria-label="Clear"
+          tooltipDirection="w"
```

Comment:
Totally nitpick and out of curiosity: Is this to hack the broken tooltip in TextInput.Action? (I created an issue for [this](https://github.com/primer/react/issues/4091))

Reply:
> Is this to hack the broken tooltip in TextInput.Action?

It is! Thank you.

Going to use Tooltip v2 in a new PR, so we might not need the issue long term